spec/models/reservation_spec.rb

```diff
@@ -102,7 +102,7 @@
   end
 
   context 'when not checked out' do
-    it { expect(reservation.status).to eq('reserved') }
+    it { expect(reservation.reserved?) }
```

Comment:
Here also, you need to `expect(reservation.reserved?).to be_truthy`, as an example.